tasklist/config/envs/.env.database.elasticsearch

```
@@ -32,6 +32,13 @@ CAMUNDA_OPERATE_ZEEBEELASTICSEARCH_URL=http://${DATABASE_CONTAINER_NAME}:9200
 CAMUNDA_OPERATE_BACKUP_REPOSITORYNAME=test
 CAMUNDA_OPERATE_DATABASE=${DATABASE}
 
+# Optimize Configuration
+OPTIMIZE_ELASTICSEARCH_HOST=${DATABASE}
+CAMUNDA_OPTIMIZE_ELASTICSEARCH_URL=http://${DATABASE_CONTAINER_NAME}:9200
+CAMUNDA_OPTIMIZE_ZEEBE_ENABLED=true
+CAMUNDA_OPTIMIZE_ZEEBE_NAME=zeebe-record
+CAMUNDA_OPTIMIZE_ZEEBE_PARTITION_COUNT=2
+
```

We can also include these as they seem defaults, and remove them from the docker-compose

```suggestion
CAMUNDA_OPTIMIZE_SECURITY_AUTH_COOKIE_SAME_SITE_ENABLED=false
CAMUNDA_OPTIMIZE_ENTERPRISE=false
CAMUNDA_OPTIMIZE_IMPORT_DATA_SKIP_DATA_AFTER_NESTED_DOC_LIMIT_REACHED=true
```